uni/lib/view/home/widgets/bus_stop_card.dart

```diff
@@ -25,10 +24,31 @@ class BusStopCard extends GenericCard {
 
   @override
   Widget buildCardContent(BuildContext context) {
-    return LazyConsumer<BusStopProvider>(
-      builder: (context, busProvider) {
-        return getCardContent(
-            context, busProvider.configuredBusStops, busProvider.status);
+    return Consumer<BusStopProvider>(
```

Comment:
Probably you had some git issues, since you are not using a `LazyConsumer` here and the data is not loaded automatically